### ISSUE TITLE
Feature Request: Canarie and LTS versions of the plugin

### ISSUE THREAD

**Issue by TheFiZi** [vrtmrz/obsidian-livesync#414]
Just wanted to throw this idea out there, release two versions of Obsidian LiveSync.

One could be the "LTS" (Long Term Support) version which is basically a known working version of the plugin that can be considered "stable".

The other could be the "Canarie" version where new features are added for testing and such. Once the new features have been tested and are working they could be rolled into the next LTS release.

A simple way to delineate this could be using version numbers. Evens (20, 22, etc) could be considered LTS and odds (21, 23) could be considered Canarie. Then within the plugin there could be a setting where you pick if you want LTS or Canarie releases and the app would filter based on that selection when you click 'Check for update'.

The general idea is for those of us who just need our notes sync'd and don't want that functionality to be impacted by having a bug introduced by work on a new feature can choose to sit on LTS and those of us who want to help you test new features can use the Canarie build.

Please do not take this as a critique on your work. I very much appreciate everything you do to develop this plugin and your quick response to any issues that do come up. I simply mean this as a general suggestion.

**Comment by vrtmrz** [comment]
Thank you for your suggestion! This is very much a good opportunity, so let me explain my thoughts.

I thought that once for the same reason. However, then I decided to try this style, at that time.

My worry is that the LTS will be neglected or that Bleeding Edge will take too many risks.

I would love to have been one of the users, as all of you (or a bit known). This means that we are using the same version, and sharing the same risk and the same performance. This means that I would not release the version that I cannot use. If it contains a indeed fatal problem, it should be fixed immediately.

As you know, as a result of this, we have very lot of switches. But we can disable all new features with these toggles if no major version has been changed[^1]. And all potentially harmful features are always disabled by default. (Of course, there may be some defects).

[^1]: So we will get a notification.

We also want to avoid the irresponsible statement that 'the latest version has already fixed it' (in this situation we would expect it to be fixed as an LTS version, but that is probably beyond our control).

But it will be better if I take into account you and other users with similar worries. I appreciate your point.
From the next release, the release note should contain a new feature with the name of the toggle to disable that, and default status.
May our work make our life productive with being relieved!
